invenio_records_lom/services/schemas/metadata.py

```diff
@@ -56,27 +74,77 @@ def _deserialize(self, value, attr, data, **kwargs):
 
         # validate inner keys
         if not isinstance(langstring_inner, dict):
-            raise self.make_error("invalid")
+            raise self.make_error("invalid_inner_type")
         inner_keys = set(langstring_inner)
         if "#text" not in inner_keys:
             raise self.make_error("missing_langstring_key", keys="#text")
-        if "lang" not in inner_keys and self.lang_exists:
+        if "lang" not in inner_keys and self.lang_exists is True:
```

Comment:
minor: why do you add the explicit check to True?

Reply:
`self.lang_exists` needs to distinguish *three* (not two) cases:
1. validate `lang` definitely exists (`lang_exists is True`)
2. validate `lang` definitely does *not* exist (`lang_exists is False`)
3. don't validate either way (`lang_exists is None`)

for three cases truthy vs falsy isn't enough...
to reflect its three-way nature, I thought about renaming it, but couldn't come up with a better name...
feel free to suggest one!

Reply:
ohhh, i see. yeah a new name is necessary. for me a variable with an prefix/postfix of `exists` is a bool value with only True/False values and None is False by definition.

where does point 3 jumps in? where is this calculated?

Reply:
resolved by using two booleans instead of a weird three-way variable